docs/Converters/ADNI2BIDS.md

```diff
@@ -69,12 +69,13 @@ Currently, the modalities supported by our converter are:
 | Florbetapir (AV45) PET (amyloid)          | -      | ✓         | ✓      |
 | Florbetaben (FBB) PET (amyloid)           | -      | -         | ✓      |
 | Flortaucipir (AV1451) PET (tau)           | -      | -         | ✓      |
+| Field mapping (FMAP)                      | ✓      | ✓         | ✓      |
 | Clinical data                             | ✓      | ✓         | ✓      |
 
 To convert the imaging data to BIDS, a list of subjects with their sessions is first obtained from the `ADNIMERGE` spreadsheet.
 This list is compared for each modality of interest to the list of scans available, as provided by modality-specific csv files (e.g. `MPRAGEMETA.csv`).
 If the modality was acquired for a specific pair of subject-session, and
-several scans and/or preprocessed images are available, only one is converted:
+several scans and/or preprocessed images are available, only one is converted except for FMAP:
```

Comment:
I'm not sure about this. Actually, I don't know what would happen if multiple images (meaning multiple full images with magnitude and phases) are available for FMAP since we are passing the parent folder to dcm2niix.
Although we never saw it in our testing datasets, I suppose this could happen if multiple runs were acquired. It would help a lot if we could find a subject in ADNI having multiple FMAP images for one session.

I think we should have a "warning" box saying that FMAP conversion was added in Clinica `0.9.0` and is still experimental. And maybe add a sentence explaining that bugs could occur especially if multiple FMAP images are available for one session.

WDYT?